pkg/config/yaml_test.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"gopkg.in/yaml.v3"
+)
+
+func TestYamlCtx(t *testing.T) {
+	data := `
```

Comment:
nit: 
1. It takes a little cognitive overhead to confirm line numbers since the data does not start on line 1. 
2. It seems odd to me that `blueprint_name` is reported as being on line 3 instead of 2. I assume it counts L26 as the first line with no characters. 

Possible ideas to reduce cognitive overhead:
- offset this data definition so that the last digit of line number matches. aka, add 5 lines of white space or comments before `data` so that L33 of the test file contains `blueprint_name` which aligns with Line 3 in mapping.
- Put data in separate file
- Do nothing

Reply:
> offset this data definition so that the last digit of line number matches.

The alignment wil break soon, as new imports will be added above;

> Put data in separate file

Data is rather small to justify the move

Added `# line D0` comments to improve navigation